python/src/CrossEntropyImportanceSampling_doc.i.in

```diff
@@ -0,0 +1,57 @@
+%feature("docstring") OT::CrossEntropyImportanceSampling
+"Cross-Entropy Importance Sampling algorithm.
+
+See also
+--------
+StandardSpaceCrossEntropyImportanceSampling, PhysicalSpaceCrossEntropyImportanceSampling, CrossEntropyResult
+
+
+Notes
+-----
+This class inherits from class:`~openturns.EventSimulation`.
+
```

Comment:
According to the code, the "CrossEntropyImportanceSampling-DefaultRhoQuantile" ResourceMap key is defined and used by the code. This should be documented in the API help page and its usage should be shown in the examples.

Reply:
Fixed.